### ISSUE TITLE
Build 1.35 for other arches?

### ISSUE THREAD

**Issue by bollwyvl** [conda-forge/ujson-feedstock#12]
Thanks for maintaining the ujson package! 

At least one downstreams _isn't_ compatible with the 2.x line, but _is_ built for aarch/ppc:

https://github.com/conda-forge/python-language-server-feedstock/pull/48

Can we get a re-rendered `1.x` branch (perhaps forked off #3) which would have those? Would PR, but it's hard to target a non-existent branch...

**Comment by dwhswenson** [comment]
It looks like the specific need from the original post was resolved by #13, but it's also a problem for Python 3.9 migration. At least, it seems to be the root of the unsatisfiability errors I'm seeing in https://github.com/conda-forge/openpathsampling-feedstock/pull/4. ([Link to Azure build](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=237474))

Our `ujson <2` pin is required because of a bug when dealing with large floats (https://github.com/ultrajson/ultrajson/issues/322, see also https://github.com/openpathsampling/openpathsampling/issues/922#issuecomment-688295920). Is there any way to have conda-forge automatically rebuild ujson 1.35 as well as the most recent release?

**Comment by bollwyvl** [comment]
I think we're good here... the bot now apparently speaks backport, so I would imagine an `osx-arm64` _should_ be forthcoming...

```
linux-64/ujson-1.35-py36h201607c_1003.tar.bz2
linux-64/ujson-1.35-py36hdce0c9c_1003.tar.bz2
linux-64/ujson-1.35-py37h745909e_1003.tar.bz2
linux-64/ujson-1.35-py38hadf7658_1003.tar.bz2
linux-64/ujson-1.35-py39h41458e0_1003.tar.bz2
linux-aarch64/ujson-1.35-py36h002c26f_1003.tar.bz2
linux-aarch64/ujson-1.35-py36hf8f5b33_1003.tar.bz2 
linux-aarch64/ujson-1.35-py37h4d2d1e1_1003.tar.bz2
linux-aarch64/ujson-1.35-py38h5ddc73f_1003.tar.bz2
linux-aarch64/ujson-1.35-py39hf9d573b_1003.tar.bz2
linux-ppc64le/ujson-1.35-py36h1a3b5e7_1003.tar.bz2
linux-ppc64le/ujson-1.35-py36h9af5095_1003.tar.bz2
linux-ppc64le/ujson-1.35-py37h761690d_1003.tar.bz2
linux-ppc64le/ujson-1.35-py38hb5f096e_1003.tar.bz2
linux-ppc64le/ujson-1.35-py39hae22bd0_1003.tar.bz2
osx-64/ujson-1.35-py36h702eb61_1003.tar.bz2
osx-64/ujson-1.35-py36haed820b_1003.tar.bz2
osx-64/ujson-1.35-py37he8cc110_1003.tar.bz2
osx-64/ujson-1.35-py38hf416894_1003.tar.bz2
osx-64/ujson-1.35-py39h7108016_1003.tar.bz2
win-64/ujson-1.35-py36he2d232f_1003.tar.bz2
win-64/ujson-1.35-py37hf2a7229_1003.tar.bz2
win-64/ujson-1.35-py38h885f38d_1003.tar.bz2
win-64/ujson-1.35-py39h415ef7b_1003.tar.bz2
```
